x509/x509_test.go

```diff
@@ -1119,6 +1119,39 @@ func TestRSAPSSSelfSigned(t *testing.T) {
 	}
 }
 
+// Valid EKCert (from a TPM)  with RSAES-OAEP Public Key.
```

Comment:
Which hardware vendor is this? Can you include that information in your comment?